File: python/type3/find_max_extra.py
# Type III variant 2 - added/removed statements
def find_max_v2(values):
    if not values:
        return None
    current_max = values[0]
    for number in values:
        if number > current_max:
            current_max = number
    print("Max found:")  # extra output
    return current_max

File: java/type3/FindMaxExtra.java
public class FindMaxExtra {
    public static Integer findMaxV2(int[] values) {
        if (values == null || values.length == 0) {
            return null;
        }
        int currentMax = values[0];
        for (int number : values) {
            if (number > currentMax) {
                currentMax = number;
            }
        }
        System.out.println("Max found:");
        return currentMax;
    }
}